scripts/run_gitleaks_dismiss.sh

```
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Bulk-dismiss gitleaks code-scanning alerts referencing files not present in the repo.
+# Usage:
+#   - Dry-run (show counts + preview): ./scripts/run_gitleaks_dismiss.sh
+#   - Execute dismissals: ./scripts/run_gitleaks_dismiss.sh --yes
+# The script reads GH_TOKEN from the environment, or falls back to parsing .env.local
+# (it will not print the token).
+
+SCRIPT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
```

Shellcheck (0.11.0)

[warning] SCRIPT_DIR appears unused. Verify use (or export if used externally).

(SC2034)